tests/tt_eager/python_api_testing/unit_testing/misc/test_create_qkv_heads.py

```diff
@@ -41,18 +43,19 @@ def run_create_qkv_heads_test(
     QKV = torch.concat([Q.flatten(-2, -1), K.flatten(-2, -1), V.flatten(-2, -1)], -1)
     QKV_interleaved = torch.concat([Q, K, V], -1).flatten(-2, -1)
 
+    ratio = num_kv_heads // cores_w
     in0_shard_spec = ttl.tensor.ShardSpec(
         ttl.tensor.CoreRangeSet(
             {
                 ttl.tensor.CoreRange(
                     ttl.tensor.CoreCoord(0, 0),
-                    ttl.tensor.CoreCoord(num_kv_heads - 1, batch - 1),
+                    ttl.tensor.CoreCoord(cores_w - 1, cores_h - 1),
                 ),
             }
         ),
         [
             seq_len,
```

Comment:
Switch this to `batch * seq_len // cores_h` for future testing? For now, you could also add an assert that `batch == cores_h`

Reply:
agreed for the future, for now I do have a batch == cores_h assert on the validate function